source/frontend/src/features/properties/map/research/add/AddResearchContainer.tsx

```diff
@@ -101,10 +102,11 @@ export const AddResearchContainer: React.FunctionComponent<
 
             <Prompt
               when={
-                formikProps.dirty ||
-                (formikProps.values.properties !== initialForm.properties &&
-                  formikProps.submitCount === 0) ||
-                (!formikProps.values.id && formikProps.values.properties.length > 0)
+                (formikProps.dirty ||
+                  (formikProps.values.properties !== initialForm.properties &&
+                    formikProps.submitCount === 0) ||
+                  (!formikProps.values.id && formikProps.values.properties.length > 0)) &&
+                !formikProps.isSubmitting
```

Comment:
**AddAcquisitionContainer** shares some of this same logic - did you check acquisitions too for this bug?